### PR TITLE
Api rewrite

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -19,7 +19,13 @@ Get Started
 ===========
 
 - New user? `Register <https://neptune.ai/register>`_ and climb aboard.
-- Registered already? Log in `here <https://neptune.ai/login>`_.
+- Registered already? Log in `here <https://neptune.ai/login>`_, then click **Getting Started** and follow the onboarding instructions:
```
Comment (kamil-kaczmarek) [inline docs/index.rst:22]: All links should be in this format: 

some text |neptune-client-github| some text.

Then at the bottom:
.. |neptune-client-github| raw:: html

    <a href="https://github.com/neptune-ai/neptune-client" target="_blank">Neptune client</a>

File: docs/python-api/introduction.rst [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-============
-|neptune-client-github| is open source Python library that allows you to integrate your Python scripts with Neptune. Neptune client supports handful of use cases:
+About neptune-client
+--------------------
 
-* creating and tracking experiments
-* managing running experiment
-* querying experiments and projects (search/download)
+|neptune-client-github|
+is an open source Python library that lets you integrate your Python scripts with Neptune so that you can more easily track and organize your experiments in the rich Neptune `dashboard <https://ui.neptune.ai/shared/onboarding/experiments>`_.
```
Comment (kamil-kaczmarek) [inline docs/python-api/introduction.rst:5]: Let's put this link: https://ui.neptune.ai/o/USERNAME/org/example-project/experiments?viewId=d7f80ebe-5bfe-4d12-97c1-2b1e6184a2ed

File: docs/python-api/introduction.rst [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 .. code:: bash
 
     pip install neptune-client
 
-Once installed, ``import neptune`` in your code to use it.
+Once installed, add ``import neptune`` to your code to use neptune-client capabilities.
 
 Example
--------
+=======
```
Comment (kamil-kaczmarek) [inline docs/python-api/introduction.rst:29]: Here is better example code, only slightly longer but presents a lot more value:


```
import neptune
import numpy as np

# select project
neptune.init('USERNAME/example-project',
             api_token='ANONYMOUS')

# define parameters
PARAMS = {'decay_factor': 0.7,
          'n_iterations': 117}

# create experiment
neptune.create_experiment(name='quick_start_example',
                          params=PARAMS)

# log some metrics
for i in range(1, PARAMS['n_iterations']):
    neptune.log_metric('iteration', i)
    neptune.log_metric('loss', PARAMS['decay_factor']/i**0.5)
    neptune.log_text('text_info', 'some value {}'.format(0.95*i**2))

# add tag to the experiment
neptune.append_tag('quick_start')

# log some images
for j in range(5):
    array = np.random.rand(10, 10, 3)*255
    array = np.repeat(array, 30, 0)
    array = np.repeat(array, 30, 1)
    neptune.log_image('mosaics', array)
```

File: docs/python-api/query-api.rst [hunk 4]
```diff
@@ -1,26 +1,35 @@
...
-* :meth:`~neptune.experiments.Experiment.get_system_properties`
-* :meth:`~neptune.experiments.Experiment.get_tags`
+Fetching Data From Neptune
+--------------------------
+
+In addition to uploading experiment data to Neptune, neptune-client can be used to fetch data directly from Neptune for further processing, such as for custom analysis.
+
+`Study the example <https://ui.neptune.ai/USERNAME/example-project/n/Experiments-analysis-with-Query-API-and-Seaborn-31510158-04e2-47a5-a823-1cd97a0d8fcd/91350522-2b98-482d-bc14-a6ff5c061b6b>`_.
+
+On the Project Level
```
Comment (kamil-kaczmarek) [inline docs/python-api/query-api.rst:10]: maybe these heading should be renamed. This exact formulation is not very informative.

File: docs/python-api/sample_project.rst [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Neptune's Sample Project
+------------------------
```
Comment (kamil-kaczmarek) [inline docs/python-api/sample_project.rst:2]: I guess this one lacks content :)

File: docs/index.rst [hunk 6]
```diff
@@ -136,11 +142,10 @@ Go ahead and mention us on social media!
    :caption: Python Library
 
    python-api/introduction.rst
-   python-api/api-reference.rst
    python-api/query-api.rst
-   python-api/tutorials.rst
-   python-api/how-to-guides.rst
+   python-api/sample_project.rst
    python-api/cheatsheet.rst
+   python-api/api-reference.rst
```
Comment (kamil-kaczmarek) [inline docs/index.rst:148]: this should be second, just below intro

File: docs/python-api/api-reference.rst [hunk 7]
```diff
@@ -1,12 +1,15 @@
-API reference
-=============
+Neptune Python Library Reference
```
Comment (kamil-kaczmarek) [inline docs/python-api/api-reference.rst:1]: On this page, links to Neptune, Session, Project, Experiment, Notebook and Utils are very important.
Maybe we should also add additional list with those links, as a quick reference or table of contents for library docs?

File: docs/index.rst (hunk 1 above)
Comment (kamil-kaczmarek) [reply docs/index.rst:22]: also links to methods

**Comment by moshekruger** [comment]
@kamil-kaczmarek Please take a look. I did a lot of work on the files.
Note that in python-api\api-reference.rst, there are three options for presenting the details:
- The original (commented out)
- The current - visible
- The one in which I attempted to use a table, as we discussed (commented out). The reason I hid it was that text in the right-hand column does not wrap (there is a known issue with this in ReadTheDocs). It makes the user have to slide back and forth to read the contents. I did not find a quick fix for this.

**Comment by moshekruger** [comment]
I see it now in the WWW. Looks very nice!

----------------------
Moshe Kruger
Mobile: +972-52-770-7387
---------------------------------------
LinkedIn Profile: www.linkedin.com/in/allwrite

**Pull request merged**